README.Rmd

```diff
@@ -8,7 +8,6 @@ bibliography: paper.bib
 ```{r, echo = FALSE}
 knitr::opts_chunk$set(
   collapse = TRUE,
-  tidy.opts = list(width.cutoff = 100),
```

Comment:
For inconsequential commits like these, please use `[skip ci]` in your commit messages (e.g. `remove tidy opts [skip ci]`). We really don't need to run 24 GHA workflows to check a change like this.